chartpress.yaml

```diff
@@ -34,3 +35,17 @@ charts:
         valuesPath: keycloak.initRealm.image
         paths:
           - scripts/init-realm
+      certificates:
+        contextPath: certificates
+        dockerfilePath: certificates/Dockerfile
+        valuesPath: global.certificates.image
+        paths:
+          - certificates
```

Comment:
I think this should be moved down to the `helm-chart/certificates` chart

Reply:
@ableuler I am not sure what you mean? I should move the `Dockerfile` so that it is located at `helm-chart/certificates`?

Reply:
So the chartpress file defines two charts, but most of the specification that should belong to the certificates chart seems to be underneath the renku chart. Anyway, as I later during the review noticed, I think there should be two separate chartpress files anyway.